README.md

```diff
@@ -1,5 +1,9 @@
 ## Hyperledger Fabric Client SDK for Node.js
 
+![](https://img.shields.io/badge/fabric--network-1.4.7-orange.svg)
```

Comment:
This is not the current release version and I really don't want to have to manually change this every release

Reply:
I aim to selectively add this to release-1.4 branch. 
I would like take the responsibility for this part.